README.md

```diff
@@ -228,16 +228,16 @@ For example:
 chmod 400 id_rsa && \
 ssh-add $(pwd)/id_rsa
 
-ssh vcap@<concourse_ip>
+ssh vcap@<deployer_concourse_ip>
```

Comment:
To clarify - I don't intend to change the ELB as part of this PR. But we could do it later on.